### ISSUE TITLE
how to predict end to end

### ISSUE THREAD

**Issue by AutumnZ-94** [jd-aig/JAVE#3]
I notice the code in` predict.py` that `cids_of_imgs  imgs.embeddedG.npy  imgs.embedded.npy  sids_of_txts  txts.embeddedG.npy  txts.embedded.npy` are required for predict and evaluation.

`       graph.get_tensor_by_name("inputs_seq:0"): inputs_seq_batch,
        graph.get_tensor_by_name("inputs_seq_len:0"): inputs_seq_len_batch,
        graph.get_tensor_by_name("inputs_seq_embedded:0"): inputs_seq_embedded_batch,
        graph.get_tensor_by_name("inputs_seq_embeddedG:0"): inputs_seq_embeddedG_batch,
        graph.get_tensor_by_name("inputs_img_embedded:0"): inputs_img_embedded_batch,
        graph.get_tensor_by_name("inputs_img_embeddedG:0"): inputs_img_embeddedG_batch,
        graph.get_tensor_by_name("dropout_prob:0"): 0`

 So how could I use it in real world to predict ? 
Whether all these input tensor are necessary without evaluation， should I switch the model to an inference model?

**Comment by wavewangyue** [comment]
Yes, all these input tensor are necessary in prediction (or evaluation)
